src/schemas/ProgramDonorSummary/resolvers/index.ts

```diff
@@ -89,7 +70,7 @@ const resolveWithProgramAuth = <ResolverType = GraphQLFieldResolver<unknown, unk
 
 const createResolvers = async (
   esClient: Client,
-): Promise<IResolvers<unknown, GlobalGqlContext>> => {
+): Promise<IResolvers<ProgramDonorSummaryStatsGqlResponse, GlobalGqlContext>> => {
```

Comment:
I poked around at the types involved in resolvers and this will be relevant if we add a resolver for a specific field instead of the entire type. This isn't needed right now but I don't want to lose this detail for if this does get modified later.